### PR TITLE
Update longhorn-ingress.md. Add missing ingressClassName to ingress .yaml

### PR DESCRIPTION
**Pull request by mrksph**
Add missing ingressClassName

#### Which issue(s) this PR fixes:

Issue #

#### Additional documentation or context

After going through Longhorn's documentation, I came across this code snippet to setup Longhorn ingress. It the text above the code snippet, it says it's using nginx as Ingress Controller but the Longhorn Ingress definition is missing the ingressClassName property, which is key to make it work.

### CODE REVIEW

**Comment by jhkrug** [comment]
Could an engineer check this please? If correct then is the change needed elsewhere, in other versions. Thanks.

**Comment by innobead** [comment]
> Could an engineer check this please? If correct then is the change needed elsewhere, in other versions. Thanks.

cc @ChanYiLin community coordinator this sprint.

cc @derekbit

**Comment by mrksph** [comment]
Hey @ChanYiLin could you take a look at this small doc update? Thanks

**Comment by mrksph** [comment]
Hey @ChanYiLin ??

**Comment by ChanYiLin** [comment]
Hi @mrksph 
Thanks for your contribution.
The change looks good to me.
But can you help to also update the files in these following version?
```
content/docs/1.6.4/deploy/accessing-the-ui/longhorn-ingress.md
content/docs/1.7.0/deploy/accessing-the-ui/longhorn-ingress.md
content/docs/1.7.1/deploy/accessing-the-ui/longhorn-ingress.md
content/docs/1.7.2/deploy/accessing-the-ui/longhorn-ingress.md (this is where you already changed)
content/docs/1.7.3/deploy/accessing-the-ui/longhorn-ingress.md
content/docs/1.8.0/deploy/accessing-the-ui/longhorn-ingress.md
```
Thanks

**Comment by mrksph** [comment]
Yes, will do

**Comment by ChanYiLin** [comment]
Hi @mrksph 
Suggest put them into one commit, thanks

**Comment by ChanYiLin** [comment]
Hi @mrksph @innobead 
When the commits are squashed, we can merge this PR.
Thanks

**Comment by mrksph** [comment]
Will do later from home, I edited this from GitHub web.

**Comment by mrksph** [comment]
Force pushed with a single commit for the rest of doc versions

**Review by ChanYiLin — APPROVED** [review]
LGTM
cc @innobead

**Review by jhkrug — DISMISSED** [review]
lgtm

File: content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Authentication is not enabled by default for kubectl and Helm installations. In
         # custom max body size for file uploading like backing image uploading
         nginx.ingress.kubernetes.io/proxy-body-size: 10000m
     spec:
+      ingressClassName: nginx
```
Comment (innobead) [inline content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md:39]: Please apply this change to the example output as well so that they align.

**Review by innobead — CHANGES_REQUESTED** [review]
Thanks for the contributioin. Just one minor change is required.

File: content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md (hunk 1 above)
Comment (mrksph) [reply content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md:39]: what is the example output sorry?

Comment (innobead) [reply content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md:39]: https://github.com/mrksph/website/blob/patch-1/content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md?plain=1#L79-L104

**Pull request merged**